CONTRIBUTING.md

```diff
@@ -143,13 +157,34 @@ Here are a few examples:
 ## Working with documentation versions
 
 The KEDA documentation is versioned. Each version has its own subdirectory under
-[content/docs](content/docs). To add a new version, copy the directory for
-the most recent version. Here's an example:
+[content/docs](content/docs). To add a new version, follow these steps:
+
+1. copy the directory for the most recent version. Here's an example:
 
 ```console
 $ cp -rf content/docs/<CurrentVersion> content/docs/<NewVersion>
 ```
 
+2. copy the file for the most recent faq data in the `data` directory. Here's an example:
+
+```console
+$ cp -rf data/faq<CurrentVersion> data/faq<NewVersion>
```

Comment:
Is there no way that the faq data is part of `content/docs/<version>`?

Reply:
unfortunately no.